src/framework/preferences/html/PrefsEditorTemplate-linksControls.html

```diff
@@ -1,4 +1,5 @@
 <h2><span class="fl-icon-links"></span><span class="flc-prefsEditor-linksControls-label heading-text"></span></h2>
+<p class="flc-prefsEditor-emphasis-descr"></p>
```

Comment:
This should probably have been `flc-prefsEditor-emphasizeLinks-descr`, but i'm not sure where it is defined/used.